README.md

```diff
@@ -95,6 +95,40 @@ const schemaValidator = {
 }
 ```
 
+## Validating using cross properties (body, params or query)
+
+The middleware puts the req in the context. Therefore it's possible to use the other properties to validate. Remember that to make use of `this` it's necessary to use a function (not an arrow one).
```

Comment:
You can cross-validate properties of your payload using a custom Yup test and accessing them by calling `this.options.context`.